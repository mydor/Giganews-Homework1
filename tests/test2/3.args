user2@domain1.com
testpw2
