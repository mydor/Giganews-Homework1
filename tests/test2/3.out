Authentication successful for user2@domain1.com
User: user2
Domain: domain1.com
Alias: 
Username: user2@domain1.com
